### PR TITLE
docs: update descriptions and add structured package data for `math/base/special/gcd`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   adds structured package data for [`math/base/special/gcd`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/gcd).

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves a part of #1147.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by gunjjoshi** [comment]
Here, we have different descriptions for the input and return values for `JS` and `C` implementations.
`JS`: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/gcd/lib/main.js#L37
`C`:  https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/gcd/src/main.c#L33

I have followed the `JS` way. But, I think a mix of both would be more appropriate, something like:

```c
a: first number/input value
b: second number/input value
returns: greatest common divisor
```

Should we change this?

**Comment by kgryte** [comment]
@gunjjoshi Yes, I think you are right. Feel free to make those changes. "first number" and "second number" seems fine to me.

File: lib/node_modules/@stdlib/math/base/special/gcd/package.json [hunk 1]
```diff
@@ -71,5 +71,129 @@
...
     "integer"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "gcd",
+      "alias": "gcd",
+      "pkg_desc": "compute the greatest common divisor (gcd)",
+      "desc": "computes greatest common divisor (gcd)",
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/gcd/package.json]:
```suggestion
      "desc": "computes the greatest common divisor (gcd)",
```

**Comment by gunjjoshi** [comment]
> @gunjjoshi Yes, I think you are right. Feel free to make those changes. "first number" and "second number" seems fine to me.

Done! Thanks.

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @gunjjoshi!

**Pull request merged**